src/openforms/authentication/contrib/digid/views.py

```diff
@@ -42,8 +42,10 @@ def get_level_of_assurance(self):
         _, _, kwargs = resolve(return_path)
         form = get_object_or_404(Form, slug=kwargs.get("slug"))
 
-        loa = form.authentication_backend_options.get(PLUGIN_ID, {}).get("loa")
-        return loa if loa else DIGID_DEFAULT_LOA
+        # called after AuthenticationStartView.get(), which already checks if there is an
+        # FormAuthenticationBackend object for the plugin
+        auth_backend = form.auth_backends.get(backend=PLUGIN_ID)
+        return auth_backend.options["loa"] or DIGID_DEFAULT_LOA
```

Comment:
ah but you can still trigger errors this way :) Try navigating directly to `http://localhost:8000/digid/login/?next=http://localhost:8000/some-test-form-slug/` in your browser!